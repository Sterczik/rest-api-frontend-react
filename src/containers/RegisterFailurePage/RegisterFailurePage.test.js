import React from 'react';
import { shallow } from 'enzyme';
import { RegisterFailurePage } from './index';

test('should render RegisterFailurePage correctly', () => {
  const wrapper = shallow(<RegisterFailurePage />);
  expect(wrapper).toMatchSnapshot();
});
