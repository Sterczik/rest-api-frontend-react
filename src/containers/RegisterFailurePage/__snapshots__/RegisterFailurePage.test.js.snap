// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render RegisterFailurePage correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <RegisterFailurePage />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": Array [
        <HelmetWrapper
          defaultTitle="Register Failure Page"
          defer={true}
          encodeSpecialCharacters={true}
          titleTemplate="Register Failure Page"
        >
          <meta
            content="Register Failure Page"
            name="description"
          />
        </HelmetWrapper>,
        <PageHeading
          title="Register Failure Page"
        />,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": <meta
            content="Register Failure Page"
            name="description"
          />,
          "defaultTitle": "Register Failure Page",
          "defer": true,
          "encodeSpecialCharacters": true,
          "titleTemplate": "Register Failure Page",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "content": "Register Failure Page",
            "name": "description",
          },
          "ref": null,
          "rendered": null,
          "type": "meta",
        },
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "title": "Register Failure Page",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": Symbol(react.fragment),
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": Array [
          <HelmetWrapper
            defaultTitle="Register Failure Page"
            defer={true}
            encodeSpecialCharacters={true}
            titleTemplate="Register Failure Page"
          >
            <meta
              content="Register Failure Page"
              name="description"
            />
          </HelmetWrapper>,
          <PageHeading
            title="Register Failure Page"
          />,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": <meta
              content="Register Failure Page"
              name="description"
            />,
            "defaultTitle": "Register Failure Page",
            "defer": true,
            "encodeSpecialCharacters": true,
            "titleTemplate": "Register Failure Page",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "content": "Register Failure Page",
              "name": "description",
            },
            "ref": null,
            "rendered": null,
            "type": "meta",
          },
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "title": "Register Failure Page",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": Symbol(react.fragment),
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
